src/components/Searchbar.js

```diff
@@ -0,0 +1,39 @@
+import styled from 'styled-components';
+
+export default function Searchbar({ handleChange, searchValue }) {
+  return (
+    <SearchbarContainer>
+      <InputLabel htmlFor="searchInput">Search for your Drink:</InputLabel>
+      <InputField
+        type="text"
+        id="searchInput"
+        name="searchInput"
+        onChange={handleChange}
+        placeholder="e.g. Caipirinha..."
+        maxLength={30}
+        value={searchValue}
+      ></InputField>
+    </SearchbarContainer>
+  );
+}
+
+const SearchbarContainer = styled.div`
```

Comment:
really nice mirco! looks quite good so far...instead of a div i would maybe declare it as a section! so it´s not wrong and you can leave it

Reply:
thanks, I will consider this for my future work :)